lib/SPIRV/SPIRVReader.cpp

```diff
@@ -562,12 +563,32 @@ void SPIRVToLLVM::setLLVMLoopMetadata(SPIRVLoopMerge *LM, BranchInst *BI) {
     Name = llvm::MDString::get(*Context, "llvm.loop.unroll.full");
   else if (LM->getLoopControl() == LoopControlDontUnrollMask)
```

Comment:
There might be multiple bits set in the loop control mask, so your implementation should support it.

Reply:
You are right, thanks!

Reply:
Done